modules/configuring-the-quickstarts/proc-starting-and-completing-modules-in-quickstarts.adoc

```diff
@@ -0,0 +1,22 @@
+[id='proc-starting-and-completing-modules-in-quickstarts_{context}']
+= Starting and completing Quickstart onboarding steps
+
+You can start and complete the Quickstart onboarding steps at your own pace.
+
+.Prerequisites
+. Your platform engineer has granted you access to the Quickstart plugin.
```

Comment:
This is not required. Instead the user should be an admin to access the quickstart in case permissions are enabled.